reference/performance/index-merge.md

```diff
@@ -0,0 +1,99 @@
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(a int, b int, c int, unique key(a), unique key(b));
+explain select * from t where a = 1 or b = 1;
+```
+
+The filter conditions in the query expression are connected by `OR`. Within the limitations of one index per table only, `a = 1` cannot be pushed down to the query `a`; neither can `b = 1` be pushed down to the query `b`. To ensure the correct result, the execution plans of `TableScan` are generated after the query:
+
+```
++---------------------+----------+-----------+------------------------------------------------------------+
+| id                  | count    | task      | operator info                                              |
++---------------------+----------+-----------+------------------------------------------------------------+
+| TableReader_7       | 8000.00  | root      | data:Selection_6                                           |
+| └─Selection_6       | 8000.00  | cop[tikv] | or(eq(test.t.a, 1), eq(test.t.b, 1))                       |
+|   └─TableScan_5     | 10000.00 | cop[tikv] | table:t, range:[-inf,+inf], keep order:false, stats:pseudo |
++---------------------+----------+-----------+------------------------------------------------------------+
+```
+
+Full table scans are inefficient in the cases of high data volume of `t`. The query only returns two rows. To handle such scenarios, TiDB introduces `IndexMerge` to access tables.
+
+## Actual use cases
+
+`IndexMerge` allows the optimizer to use multiple indexes per table, and merge the results returned by the indexes before further operation.Take the query above as an example, the execution plans generated will be:
+
+```
++--------------------+-------+-----------+---------------------------------------------------------------+
+| id                 | count | task      | operator info                                                 |
++--------------------+-------+-----------+---------------------------------------------------------------+
+| IndexMerge_11      | 2.00  | root      |                                                               |
+| ├─IndexScan_8      | 1.00  | cop[tikv] | table:t, index:a, range:[1,1], keep order:false, stats:pseudo |
+| ├─IndexScan_9      | 1.00  | cop[tikv] | table:t, index:b, range:[1,1], keep order:false, stats:pseudo |
+| └─TableScan_10     | 2.00  | cop[tikv] | table:t, keep order:false, stats:pseudo                       |
++--------------------+-------+-----------+---------------------------------------------------------------+
+```
+
+The structure of the `IndexMerge` execution plan is similar to that of the `IndexLookUp` since both can be divided into index scans and full table scans. However, index scans of the `IndexMerge` may include multiple `IndexScan`. When the primary key index of the table is an integer, index scans may include one `TableScan`. For example:
```

Comment:
```suggestion
The structure of the `IndexMerge` execution plan is similar to that of the `IndexLookUp` since both are composed of index scans and table scans. However, index scan part of the `IndexMerge` may include multiple `IndexScan`. When the primary key of the table is an integer, index scans may even include one `TableScan`. For example:
```